lib/pages/PublisherDetails/PublisherDetails.tsx

```diff
@@ -0,0 +1,231 @@
+    dispatch(getPublisherDetailsAction(publisherID))
+  }, [publisherID])
+
+  const [publisherNameInitials, setPublisherNameInitials] = useState('...')
+
+  useEffect(() => {
+    const publisherNameInitialsArray = publisherDetails.name.split(' ')
+    const publisherNameInitials =
+      publisherNameInitialsArray.length >= 2
+        ? `${publisherNameInitialsArray[0][0]}${publisherNameInitialsArray[1][0]}`
+        : `${publisherNameInitialsArray[0][0]}${publisherNameInitialsArray[0][1]}`
+
+    setPublisherNameInitials(publisherNameInitials)
+  }, [publisherDetails])
+
+  const generatePublisherLinks = (providedPublisherLinks) => {
+    const publisherLinks = []
+
+    for (const link in providedPublisherLinks) {
+      if (providedPublisherLinks[link]) {
+        publisherLinks.push(
+          <Grid item key={link}>
+            <Link to={providedPublisherLinks[link]}>
+              <Typography
+                style={{
+                  color: palette.text.secondary,
+                  textDecoration: 'underline',
+                }}
+                variant="body1"
+              >
+                {t(`publisherDetails.${link}`)}
+              </Typography>
+            </Link>
+          </Grid>
+        )
+      }
+    }
+
+    return (
+      <Grid
+        alignItems="flex-start"
+        container
+        direction="row"
+        justify="flex-start"
+        spacing={3}
+      >
+        {...publisherLinks}
+      </Grid>
+    )
+  }
+
+  // Pagination logic
+  const [currentPage, setCurrentPage] = useState(1)
+
+  const handleChange = (event, value) => {
+    setCurrentPage(value)
+  }
+
+  const generatePagination = () => {
+    const amountOfPages = Math.ceil(
+      pagination.rowCount / PUBLISHER_APPS_PER_PAGE
+    )
+
+    return (
+      <Pagination
+        color="primary"
+        count={amountOfPages || 1}
+        onChange={handleChange}
+        page={currentPage}
+        shape="rounded"
+      />
+    )
+  }
+
+  useEffect(() => {
+    /* Triggers the retrieval and storage (under the 'marketplace' section of our app's Store)
+    of all apps a particular publisher has made available on the Marketplace.
+    
+    If the user changes pages by way of the pagination, we need to do this again. */
+    if (retrievedPublisherDetails) {
+      dispatch(
+        getFilteredMarketplaceAppsAction(
+          {
+            label: [],
+            order: 'desc',
+            org_id: [publisherID],
+            page: 1,
+            pageSize: 12,
+            search: '',
+            sort_by: 'app',
+          },
+          'publisher'
+        )
+      )
+    }
+  }, [currentPage, retrievedPublisherDetails])
+
+  return (
+    <Container className={classes.pageContainer}>
+      {/* Publisher card */}
+      <Grid
+        alignItems="center"
+        className={classes.publisherCard}
+        container
+        direction="row"
+        justify="flex-start"
+      >
+        <Grid item>
+          <Avatar
```

Comment:
If the image is transparent, yes.

Reply:
Confirm if that's the expected behaviour (gradient on transparent image)

Reply:
By request of the designers, I've added a checkered background to the Avatar, making it so that transparent images show this checkered background.

I don't necessarily agree with this, but I understand your reasoning (it isn't supposed to be the gradient after all) as well as the designer's (solid color out of question, transparent Avatar as well because we lose the Avatar's outlining and we completely lose contrast).

Reply:
(Image for reference)

<img width="245" alt="Screenshot 2021-07-06 at 17 55 17" src="https://user-images.githubusercontent.com/67902809/124640969-1426c080-de86-11eb-917f-c826255ead1e.png">

Reply:
Image for reference:

<img width="226" alt="Screenshot 2021-07-06 at 18 15 31" src="https://user-images.githubusercontent.com/67902809/124641046-2dc80800-de86-11eb-9c95-5a36f4fe0188.png">